### ISSUE TITLE
When will this be updated for 21.10?

### ISSUE THREAD

**Issue by walking-octopus** [Jannomag/Yaru-Colors#129]
In #124 and README file, GNOME 40 support was planned for the release of Ubuntu 21.10. When will anyone update this with the latest stylesheets?

**Comment by Scotty-Trees** [comment]
Greetings, I'm not one of the devs but they have said before that life was going to be really busy for them for a while so things might slow down a bit. The last commit was back in Oct, and this is all volunteer work, so I think they can only get around to it when they have the time and there just isn't much of it for them lately. It's unfortunate, I love Yaru Colors, but I'm on Gnome 41 now and it's not compatible, so I use yaru-gtk-theme for the time being. I do hope they can get a new release soon, but life, family, friends, etc come first, so for now I just wish the devs well and hope they have some happy holidays.

**Comment by walking-octopus** [comment]
Of course, I was just wondering if any volunteer or the author can update it. If there's any documentation on how Yaru source code is processed, someone could help with this.

**Comment by koplo199** [comment]
@walking-octopus As a workaround you can use my [branch](https://github.com/koplo199/Yaru-Colors), it is compatible with 21.10, it fix #124 #122 #120 plus some other bugs, and have a new Yaru variant based on the [Dracula](https://draculatheme.com/) color-scheme.

This is still WIP as I'm not yet 100% satisfied with the result of the Dracula variant (and more generally not satisfied by the absolute mess that is this update, especially style consistency wise), but it should be fine for other variants as there shouldn't be any difference compared to upstream Yaru.

Feel free to report any bug though. 
A known issue is GDM3 incompatibility for now (the incriminated file is `gnome-shell-theme.gresource`), but it should be resolved soon enough.

**Comment by Muqtxdir** [comment]
hi @walking-octopus @Scotty-Trees 

**Yaru-colors** is now packaged for [`what -if PPA`]( https://launchpad.net/~muqtxdir-m/+archive/ubuntu/what-if) for ubuntu 21.10.

#### install:
```bash
sudo add-apt-repository ppa:muqtxdir-m/what-if
sudo apt update
sudo apt upgrade
```

#### uninstall:
```bash
sudo apt install ppa-purge && sudo ppa-purge ppa:muqtxdir-m/what-if
```
